tex/introduction.tex

```diff
@@ -15,7 +15,54 @@ \section{Introduction}
+\begin{itemize}
+  \item \textbf{Remote access:}  Users want to operate a personal computer and have a way of starting execution on an HPC machine, capturing the results to be displayed on their personal computer.
+  \item \textbf{Minimal software requirements on the user side:}  The computers users can access directly are very diverse.  Most of them, however, do have a web browser.
+  \item \textbf{Web based software on the server side:}  Most computational notebook tools provide a built-in\footnote{For others there may be a way of using, e.g., https://novnc.com/info.html.} way of directly connecting to them via HTTP/HTTPS.
+  \item \textbf{Ability to chose arbitrary resources:}  The resource requirements range from very low memory and CPU demands that could easily be fulfilled on a shared machine to requiring multiple HPC nodes or special hardware like GPUs.
+\end{itemize}
+
+A typical JupyterHub-based session in an HPC centre may work as follows:
+\begin{itemize}
+  \item \textbf{Authentication:} The user logs in to a JupyterHub that provides different choices for the actual Jupyter session.
+  \item \textbf{Spawning of a server:} The user choses how to work (resources, duration, machine, account, project etc.) and the JupyterHub starts spawning a Jupyter user server on the selected HPC resources.
+  This may take time (seconds to hours / days).
+  \item \textbf{Connection to the user server:} When the user server is ready, the user is connected or provided with a link to connect to the user server.
+  The web browser downloads the Jupyter software and starts the Jupyter interface on the user's computer.
+  \item \textbf{Starting a kernel:} After possibly using other ways of working with the Jupyter user server\footnote{There is, e.g., a basic file system browser, a text editor, a terminal, etc. built into Jupyter.}, the user opens a notebook and explicitly or implicitly choses and starts a Jupyter kernel process which will execute the code entered by the user.
+  The user may open more than one notebook and work with more than one kernel process.
+  Kernels can be stopped and restarted by the user.
+  \item \textbf{Exporting and downloading the notebook:} When done with all or parts of the desired work, the user may export one or more notebooks to different formats (including the native notebook format) and download them to their personal computer.
+  \item \textbf{Stopping the user server:} The user server can be stopped by the user (from within the Jupyter server or from the JupyterHub), by the HPC resource manager (because of time or other resource limit violation), or by the JupyterHub.
+  \item \textbf{Disconnecting:} The user disconnects from the Jupyter user server by, e.g., closing the browser tab.
+  In a next or in the same step, the user also could disconnect from or log out of the JupyterHub.
```

Comment:
Maybe we can filter out the hub specific requirements and merge this with the user requirements section above? How the JupyterHub works in general should be briefly stated in the introduction somehow, though.